CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 #### 2.2.6
 
 * [Laravel5][Lumen] Fixed issue that caused the `have` and `haveMultiple` methods not being available when using the ORM part of the modules. See #3587. By @janhenkgerritsen
+* [InnerBrowser] Fixed clicking on a button inside the link
```

Comment:
Next time please mention this as [PhpBrowser][Frameworks], because InnerBrowser is too inner for others to know about it :)